tools/pyprophet/pyprophet_export.xml

```diff
@@ -88,12 +88,32 @@ data.annotated <- sample_annotation(data_me, study_design, column.file = "filena
 ## remove decoys when generating plots
 data.annotated.nodecoy <- subset(data.annotated, decoy==FALSE)
 
-pdf("summary.pdf", fonts = "Times", pointsize = 12)
+pdf("summary.pdf", fonts = "Times", pointsize = 8)
 plot(0,type='n',axes=FALSE,ann=FALSE)
 title(main="Summarized plots and tables from pyprophet export file")
 
 ## Look at Numbers of peptides and proteins per run
-grid.table(count_analytes(data.annotated.nodecoy), rows= NULL)
+## for many runs table needs to be split over several pages
+number_samples = nrow(count_analytes(data.annotated.nodecoy))
+
+    ### for more than 20 annotation groups print only 20 samples per page:
+    if (number_samples <= 20){
+        grid.table(count_analytes(data.annotated.nodecoy), rows= NULL)
+    }else{
+        grid.table(count_analytes(data.annotated.nodecoy)[1:20,], rows= NULL)
+        mincount = 21
+        maxcount = 40
+        for (count15 in 1:(ceiling(number_samples/20)-1)){
```

Comment:
Meli it is better to add a test for this if at all possible, otherwise such a logic can miss pieces.

Reply:
unfortunately no chance to use a test file with > 20 samples